de.metas.adempiere.adempiere/client/src/main/java/org/compiere/grid/ed/VLookupAutoCompleterValidationRule.java

```diff
@@ -89,10 +91,22 @@ public IStringExpression getPrefilterWhereClause()
 		}
 		return params;
 	}
-	
+
 	@Override
 	public INamePairPredicate getPostQueryFilter()
 	{
 		return INamePairPredicate.NULL;
 	}
+
+	@Override
+	public List<ValueNamePair> getExceptionTableAndColumns()
+	{
+		return ImmutableList.of();
+	}
+
+	@Override
+	public void registerException(final String tableName, final String columnName)
+	{
+		throw new NotImplementedException("There is no implementation for registering esceptions in the VLookupAutoCompleterValidationRule class: " + this);
```

Comment:
briefly thought with teo about this: `NotImplementedException` is a much nicer classname that `UnSupportedOperationException`, but `UnSupportedOperationException` is in the `java.lang` package and i'm not sure we want to further bind ourselves to commons-lang3..

Reply:
Done

Reply:
Done